Source/src/Modules/ModuleInput.h

```diff
@@ -63,7 +63,7 @@ namespace Hachiko
 
         KeyState* keyboard = nullptr;
         Uint32 mouse{};
-        SDL_Keymod keymods;
+        SDL_Keymod keymods{};
```

Comment:
We should have all the initialization of the member fields in the default constructor in the order of their declarations. In the next iteration maybe we can have that consitency update in all classes.